packages/app/src/features/compliance/ComplianceContainer.tsx

```diff
@@ -12,7 +12,13 @@ export function ComplianceContainer() {
 
   return (
     <Dialog open={visibleModal.type !== 'none'}>
-      <DialogContent showCloseButton={false}>
+      <DialogContent
+        showCloseButton={false}
+        // https://github.com/radix-ui/primitives/issues/2248
```

Comment:
Linking to issue is good but also we should try to describe the issue in a sentence like:
```suggestion
        // Prevents tooltip open automatically https://github.com/radix-ui/primitives/issues/2248
```